src/main/java/org/troyargonauts/subsystems/Manipulator.java

```diff
@@ -0,0 +1,53 @@
+package org.troyargonauts.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.troyargonauts.Constants.ManipulatorConstants;
+
+/**
+ * runs 1 motor to run manipulator
```

Comment:
Full sentences and proper capitalization please. Where is manipulator attached? Answer in Javadocs.